### ISSUE TITLE
Significant lag while editing order cycles

### ISSUE THREAD

**Issue by RonellaG** [openfoodfoundation/openfoodnetwork#5926]
## Description

One manager from Melbourne Farmers Marketplace noticed some significant lag issues with OFN today and last night.  Editing order cycles was taking a long time but he had no issues with internet speeds (45 down/5 up from a speedtest). Another MFM manager in a different location was experiencing a similar issue from her place in Gippsland.

## Expected Behavior

The load time for each screen within Edit Order Cycle should be fast.

## Actual Behaviour

One manager said the load time for each screen within Edit Order Cycle is around 3-4 minutes - and has been like that since last night.

The other manager experienced about 1-2 minutes per action. She said: 
press the edit button (wait), general settings page comes up (wait), fields eventually populate, press incoming products page (wait)... etc  So there's a lag on every action (loading... saving... etc)

## Steps to Reproduce

Logged in as hello+ronella - editing the Macedon Buying Collective order cycle
press the edit button - general settings page comes up (30 seconds), 
press Incoming Products page (38 secs)
fields eventually populate (30 seconds), 
press Outgoing Products page (30 secs)
outgoing products fields eventually populate (20 secs)
There is definitely an unusually long lag for each step.

## Your Environment

* Version used:
* Browser name and version: Google Chrome Version 84.0.4147.125 (Official Build) (64-bit)
* Operating System and version (desktop or mobile): MacOS Catalina Version 10.15.6 on laptop

**Comment by RonellaG** [comment]
The customer reported that Supplier reports was laggy as well.

**Comment by lin-d-hop** [comment]
Old performance issues are hard to follow up on. Closing
